.github/workflows/audit.yaml

```diff
@@ -31,21 +31,25 @@ jobs:
       - run: |
           for file in $(find -name requirements.txt)
           do
-            (cd $(dirname ${file}) && safety check --full-report --file=requirements.txt)
+            echo Audit ${file}
+            (cd $(dirname ${file}) && safety check --full-report --ignore=$(cat safety-ignore|true) \
```

Comment:
Seems --ignore can be specified many times, but should not support comma separated list of ids.

Reply:
https://github.com/pyupio/safety#--ignore--i

Reply:
Yes, in the safety-ignore I should add comma separated list of ids :-)